docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -27,7 +27,7 @@ This article assumes that you know how to use a command line and have a preferre
 1. Open a Command Line/Terminal.
```

Comment:
@spboyer it's ok to use when talking in the context of the tooling. I thought just that step could be wrong. You can go to http://style to see more info on that.